### ISSUE TITLE
'ScrollableState' is missing implementations

### ISSUE THREAD

**Issue by shubhamsinghshubham777** [hgraceb/nested_scroll_views#3]
Hi @hgraceb, I'm seeing the following error when trying out your library. Could you please help?

```
../../.pub-cache/hosted/pub.dev/nested_scroll_views-0.0.7/lib/src/flutter/widgets/scrollable.dart:367:7: Error: The non-abstract class 'ScrollableState' is missing implementations for these members:
 - ScrollContext.devicePixelRatio
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, RestorationMixin
      ^^^^^^^^^^^^^^^
../../fvm/versions/3.12.0/packages/flutter/lib/src/widgets/scroll_context.dart:47:14: Context: 'ScrollContext.devicePixelRatio' is defined here.
  double get devicePixelRatio;
             ^^^^^^^^^^^^^^^^
../../.pub-cache/hosted/pub.dev/nested_scroll_views-0.0.7/lib/src/nested/widgets/scrollable.dart:57:7: Error: The non-abstract class '_OverscrollScrollableState' is missing implementations for these members:
 - ScrollContext.devicePixelRatio
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class _OverscrollScrollableState extends ScrollableState {
      ^^^^^^^^^^^^^^^^^^^^^^^^^^
../../fvm/versions/3.12.0/packages/flutter/lib/src/widgets/scroll_context.dart:47:14: Context: 'ScrollContext.devicePixelRatio' is defined here.
  double get devicePixelRatio;
             ^^^^^^^^^^^^^^^^
Restarted application in 2,113ms.

```

**Comment by shubhamsinghshubham777** [comment]
Turns out I was using the beta version of Flutter i.e. `3.12.0`, downgrading to the latest stable version i.e. `3.10.5` made the error go away 👍🏻 

But nevertheless, this issue/ticket can be taken up for the next version of this library i.e. to `Support newer Flutter versions`.

**Comment by kfranic1** [comment]
The issue is now on stable channel as of version `3.13.0`

**Comment by kfranic1** [comment]
I have created a [pull request](https://github.com/hgraceb/nested_scroll_views/pull/4) for the issue.

**Comment by hgraceb** [comment]
> I have created a [pull request](https://github.com/hgraceb/nested_scroll_views/pull/4) for the issue.

Thank you very much. The issue has been fixed in version 0.0.8.
